code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm

```diff
@@ -8,7 +8,7 @@
 		SPECIES_SERGAL			= 'icons/mob/species/sergal/helmet_vr.dmi',
 		SPECIES_ZORREN_FLAT		= 'icons/mob/species/fennec/helmet_vr.dmi',
 		SPECIES_ZORREN_HIGH 	= 'icons/mob/species/fox/helmet_vr.dmi',
-		SPECIES_VULPKANI 		= 'icons/mob/species/vulpkanin/helmet.dmi',
+		SPECIES_VULPKANIN 		= 'icons/mob/species/vulpkanin/helmet.dmi',
```

Comment:
💢 Was it really that badly overlooked for this define here??? Good grief!

Reply:
Yep! I fixed this on Cit back in July last year.